HISTORY.md

```diff
@@ -6,7 +6,7 @@
 
 #### API
 
-- Add GradientOperatorEmitter to support OMG-MEGA and OG-MAP-Elites ({pr}`348`)
+- Add Base Operator Interface and Emitter Operator Retrieval
```

Comment:
Can you add the PR number to this? Also, set the PR title to be the same as this line (my idea is to keep HISTORY in line with the PR/commit history).